packages/dependency-profiles/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@fluentui-react-native/dependency-profiles",
+  "version": "0.0.0",
+  "description": "@rnx-kit/dep-check profiles covering packages published from FluentUI-React-Native",
```

Comment:
Just a heads up: `@rnx-kit/dep-check` is being renamed to `@rnx-kit/align-deps`. This package will still work as is.